src/Engine/ExpressionContainerInterface.php

```diff
@@ -48,4 +48,12 @@ public function addExpression($plugin_id, ContextConfig $config = NULL);
    */
   public function addExpressionObject(ExpressionInterface $expression);
 
+  /**
+   * Deletes an element indentified by the specified UUID in the container.
```

Comment:
expression. Maybe removing from the container would be more suiting than "deleting" ?